### DISCUSSION TITLE
Google Pay and Sepa Debit Options is not showing in Payment Sheet

### DISCUSSION THREAD

**Discussion by lohitbura07** [flutter-stripe/flutter_stripe#968] [Q&A]
**Describe the bug**
Google Pay and Sepa Debit Options is not showing in Payment Sheet

**To Reproduce**

 await Stripe.instance.initPaymentSheet(
                    paymentSheetParameters: SetupPaymentSheetParameters(
                      allowsDelayedPaymentMethods: true,
                  merchantDisplayName: 'MRGO',
                  setupIntentClientSecret: setupIntentModel.setupIntent?.first,
                  googlePay: const PaymentSheetGooglePay(
                      merchantCountryCode: 'DE',
                      testEnv: true,
                      currencyCode: 'EUR'),
                ));

await Stripe.instance.presentPaymentSheet();

**Expected behavior**
Google Pay and Sepa Debit should also show on payment sheet . But I am only getting Card 

**Smartphone / tablet**
 - Device: Pixel 4,  Api 33
 - Package version: 5.0.0
 - Flutter version: 3.3.0

bool value =  await Stripe.instance.isGooglePaySupported(IsGooglePaySupportedParams(
                testEnv: true
              ));
This value is Coming True.

**Comment by Albertbol** [comment]
Same here I test on android and get false when running:
```
final googlePaySupported =
      await Stripe.instance.isGooglePaySupported(IsGooglePaySupportedParams()); // false
```
Is something specific need to be set up to support it? In the stripe dashboard, it's turned on.

**Comment by Albertbol** [comment]
My problem was that I did not have https://pub.dev/packages/pay installed.

**Comment by lohitbura07** [comment]
@Albertbol  Can you share your method and all configuration and step you followed please . Because I just added this Library and nothing happened.  I have Enabled the Google pay option from Stripe too.

**Comment by remonh87** [comment]
The google pay plugin should not be needed. You can use the native Stripe embedding but you need to make sure that you add the gms wallet as instructed in the [docs](https://docs.page/flutter-stripe/flutter_stripe/sheet#7-optional---enable-google-pay-only-android)

**Comment by lohitbura07** [comment]
@remonh87 I am doing same as docs but still getting this. Is it something related to my current location . I am using DE (Germany) as Country Code and EUR as Currency Code but My Current Location is India. Though I have also used IN (India) and INR too but nothing happened. Also Why I am not getting the SEPA Option? Can you please help , I am stuck in this for over 2 weeks now

**Comment by lohitbura07** [comment]
@remonh87 Any Update Regarding this ? Please let me know.

**Reply by remonh87** [reply]
Did you enable the SEPA on your Stripe dashboard? Note that some payment methods are not available because it requires the country of residence to be European union (not sure if this is for sepa). I am able to get the google pay plugin in the example app.

**Reply by 10terabyte** [reply]
paymentSheetParameters: SetupPaymentSheetParameters(
    paymentIntentClientSecret: data['client_secret'], //Gotten from payment intent
    style: ThemeMode.light,
    googlePay: const PaymentSheetGooglePay(merchantCountryCode: "GB", currencyCode: "GBP", testEnv: true),
    merchantDisplayName: '${appProvider.saveBookingRequest?.bookedPersonName}'
)

This is my code to init paymentsheet.
Enabled google pay in stripe dashboard already.
but I cannot see google pay button.
Please let me know how to do it.

**Reply by remonh87** [reply]
also make sure your phone has valid google pay else it won't be shown. I would recommend contacting stripe support about this since it works on my phone.

**Reply by MuhammadMusa22** [reply]
I am facing this issue even in the latest version and i am targeting Norwegian Country with Currency NOK, and i am integrating this in Pakistan where no services are available and the funny part is, that only on one mobile Google Pay button is showing and on all other devices Google pay button is not available. Need help if someone have solution for this.
